frontend/src/components/RoutePlanner/RoutePlanner.module.css

```diff
@@ -0,0 +1,93 @@
+ion-modal#favorite_dialogue {
+  --height: 25%;
+  --width: 90%;
+  --border-radius: 16px;
+  --box-shadow: 0 10px 15px -3px rgb(0 0 0 / 0.1), 0 4px 6px -4px rgb(0 0 0 / 0.1);
+}
+
+ion-modal#favorite_dialogue::part(backdrop) {
+  background: rgba(209, 213, 219);
+  opacity: 1;
+}
+
+ion-modal#favorite_dialogue ion-toolbar {
+  --background: rgb(14 116 144);
+  --color: white;
+}
+
+#content_section{
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  text-align: center;
+
+  #buttons{
+    margin-top: 14px;
+    justify-content: space-around;
+    display: flex;
+    width: 100%;
+
+    ion-button {
+      --width: 20px;
+    }
+  }
+}
+
+.date_time_card{
+  background-color: transparent;
+  /* Adjust with global colors */
+  border: 1px solid darkslategray;
+  border-radius: 10px;
+  margin-inline-start: 4%;
+  margin-inline-end: 4%;
+}
+
+.toggle_button{
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
+}
+
+.toggle_button_row{
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding-bottom: 0.5rem;
+}
+
+.toggle_button_label{
+  align-self: center;
+}
+
+.date_time_button {
+  color: primary;
+}
+
+.date_icon {
+  font-size: 4vh;
```

Comment:
don't use vh as a font-size. use rem.

Reply:
I'm not even sure vh is working with fonts.

Reply:
It does work, but changed it to rem.